src/character_attire.cpp

```diff
@@ -1860,7 +1855,7 @@ void outfit::absorb_damage( Character &guy, damage_unit &elem, bodypart_id bp,
     // if this body part has sub part locations roll one
     if( !bp->sub_parts.empty() ) {
         sbp = bp->random_sub_part( false );
-        // the torso and legs has a second layer of hanging body parts
+        // the torso nad legs has a second layer of hanging body parts
```

Comment:
```suggestion
        // the torso and legs has a second layer of hanging body parts
```

Reply:
this should go in https://github.com/CleverRaven/Cataclysm-DDA/pull/72169